mesop/dataclass_utils/dataclass_utils.py

```diff
@@ -149,6 +154,10 @@ def decode_mesop_json_state_hook(dct):
 
     if _PANDAS_OBJECT_KEY in dct:
       return pd.read_json(StringIO(dct[_PANDAS_OBJECT_KEY]), orient="table")
+  
+  if _BYTES_OBJECT_KEY in dct:
+    return base64.b64decode(dct[_BYTES_OBJECT_KEY]) # Decode bytes
```

Comment:
[nit] Same here. We can assume the reader of the code can read what the code is doing. Usually you want to add comments that are not self-explanatory. For example, the reason why something is being done which may not be intuitive to the reader of the code.